release-ftp.sh

```diff
@@ -64,6 +64,9 @@ else
   cp daily/qsub-$TODAY.zip            /home/common/matlab/fieldtrip/data/ftp/modules
   cp daily/buffer-$TODAY.zip          /home/common/matlab/fieldtrip/data/ftp/modules
 
+  # Tweet about it using (sendweet.sh)
+  ./sendtweet "A New fieldtrip version "$TODAY" is out! Make sure you have our latest and finest :) "
```

Comment:
Should we include information about the changes? Or a download link? 

What do the @fieldtrip/maintainers think of this?